changelog.txt

```diff
@@ -24,6 +24,11 @@ Deprecated
  - ``index`` argument is deprecated in ``Project`` methods (#602, #588).
  - ``signac.cite`` module is deprecated (#611, #592).
 
+Changed
++++++++
+
+ - Project names have a default in anticipation of removing names entirely.
```

Comment:
I think this changelog can be clearer about our intentions:
```suggestion
 - Project names have a default value of ``'project'``. Project names will no longer be used in signac 2.0.
```

I considered the wording "Project names will be removed in signac 2.0." In my recollection, we're not going to delete the key if it already exists, it will just be ignored. If we're deleting the project name key as part of the schema migration, we might consider "will be removed." Otherwise "will no longer be used" feels most appropriate to me.

Reply:
We are as discussed in https://github.com/glotzerlab/signac/pull/643#discussion_r776499851, so I can use that wording.